test/util/upgrade.js

```diff
@@ -962,15 +965,15 @@ async function getMetaTxContractState() {
 
 async function getMetaTxPrivateContractState(protocolDiamondAddress) {
   /*
-    ProtocolMetaTxInfo storage layout
-
-    #0 [ currentSenderAddress + isMetaTransaction ]
-    #1 [ domain separator ]
-    #2 [ ] // placeholder for usedNonce
-    #3 [ cachedChainId ]
-    #4 [ ] // placeholder for inputType
-    #5 [ ] // placeholder for hashInfo
-    */
+      ProtocolMetaTxInfo storage layout
+  
+      #0 [ currentSenderAddress + isMetaTransaction ]
+      #1 [ domain separator ]
+      #2 [ ] // placeholder for usedNonce
+      #3 [ cachedChainId ]
+      #4 [ ] // placeholder for inputType
+      #5 [ ] // placeholder for hashInfo
+      */
```

Comment:
`ProtocolMetaTxInfo` has an additional field (`isAllowlisted`) that should be tested during this upgrade, to be correctly changed).

Reply:
I believe we would need a specific test script that we call setAllowlistedFunctions to remove old functions (like old updateSeller function id in this case) and add new ids.